### PR TITLE
added qonsoll/react-design and removed bootstrap

### PR DESCRIPTION
**Pull request by sashka2131**

### CODE REVIEW

**Review by 5P4C3GR33D — APPROVED** [review]

File: src/app/components/Lib/Modal/Modal.template.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <Typography>{title}</Typography>
+      </DialogTitle>
+      <DialogContent>{children}</DialogContent>
+      <DialogActions>
+        <Button
+          onClick={onCancel}
+          variant="contained"
+          size="small"
+          color="primary">
+          Cancel
```
Comment (alexpassss) [inline src/app/components/Lib/Modal/Modal.template.js:35]: Do not use hardcoded values here

File: src/app/components/Lib/Modal/Modal.template.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        </Button>
+        <Button
+          onClick={onOk}
+          variant="contained"
+          size="small"
+          color="secondary"
+          type="submit">
+          Save
```
Comment (alexpassss) [inline src/app/components/Lib/Modal/Modal.template.js:43]: Same here

File: src/app/components/Lib/Modal/Modal.template.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  return (
+    <Dialog open={open} aria-labelledby="form-dialog-title">
+      <DialogTitle>
+        <Typography>{title}</Typography>
+      </DialogTitle>
+      <DialogContent>{children}</DialogContent>
+      <DialogActions>
+        <Button
+          onClick={onCancel}
+          variant="contained"
```
Comment (alexpassss) [inline src/app/components/Lib/Modal/Modal.template.js:32]: Should be possible to change button style outside the component

File: src/app/components/Lib/Modal/Modal.template.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          color="secondary"
+          type="submit">
+          Save
+        </Button>
+      </DialogActions>
+    </Dialog>
+  )
+}
+Modal.propTypes = {
+  title: PropTypes.string
```
Comment (alexpassss) [inline src/app/components/Lib/Modal/Modal.template.js:50]: what about children?

File: src/app/components/Lib/Modal/Modal.template.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+}
+Modal.defaultProps = {}
```
Comment (alexpassss) [inline src/app/components/Lib/Modal/Modal.template.js:52]: remove default props

File: src/app/components/Lib/Modal/Modal.template.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+const Modal = (props) => {
+  const {
+    open,
+    setOpen,
+    title,
+    children,
+    onClick,
+    onChange,
+    onOk,
```
Comment (alexpassss) [inline src/app/components/Lib/Modal/Modal.template.js:19]: onOk what does it mean? Issues with naming

**Review by alexpassss — CHANGES_REQUESTED** [review]

**Pull request closed without merging**